roles/st2repos/tasks/redhat.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Add ST2 Repo
+  become: yes
+  yum_repository:
+    name: "StackStorm_{{ st2_pkg_repo }}"
+    description: "StackStorm_{{ st2_pkg_repo }}"
+    file: st2
+    baseurl: https://packagecloud.io/StackStorm/{{ st2_pkg_repo }}/el/{{ ansible_distribution_major_version }}/$basearch
```

Comment:
So seems this is good enough and doesn't look as hack anymore :+1: 

I think we can close the https://github.com/StackStorm/ansible-st2/issues/89 now, just add a note that we should always use `{{ ansible_distribution_major_version }}` instead of `$releasever` when adding _any_ yum repo.